pages/common/ranger.md

```diff
@@ -0,0 +1,24 @@
+# ranger
+
+> Console file manager with VI key bindings.
+> Homepage: <https://github.com/ranger/ranger>.
+
+- Open up file manager:
+
+`ranger`
+
+- Show only directories, no files or links:
+
+`ranger --show-only-dirs`
+
+- Change the configuration directory:
+
+`ranger --confdir={{dir}}`
```

Comment:
Could we change `dir` to be `path/to/directory` please? This will keep it consistent with other pages.

Same with the example below this one.

Reply:
No problem. It looks better.